Form components and CSS update

DaveyDevs opened the pull request.

laredotornado commented on client/src/components/EntityTypes.jsx:
```diff
@@ -0,0 +1,49 @@
+	);
+};
+const inputStyle = {
+	color: '#124E54'
```
Can we move this into a CSS file and reference a class from there?

DaveyDevs replied:
Done!

DaveyDevs commented:
- I was able to remove the EntityTypes component and add the "multiple" functionality to the DropDownInput component.
- Updated serializers.py to reference "contact" instead of "organization" for email and phone. This allowed it to post to the google sheet. 
- Deleted in inputStyle reference in new components and added to CSS.
